packages/ckeditor5-html-embed/src/htmlembedcommand.ts

```diff
@@ -91,7 +90,7 @@ function isHtmlEmbedAllowedInParent( selection: DocumentSelection, schema: Schem
  * Returns a node that will be used to insert a html embed with `model.insertContent` to check if a html embed element can be placed there.
  */
 function getInsertHtmlEmbedParent( selection: Selection | DocumentSelection, model: Model ): Element {
-	const insertionRange = findOptimalInsertionRange( selection, model );
+	const insertionRange = model.schema.findOptimalInsertionRange( selection, model );
```

Comment:
This was using the widget implementation of this method. I think this should not change.

Reply:
Adressed in 24ac1fa